flocker/node/agents/functional/test_cinder.py

```diff
@@ -335,7 +329,9 @@ def test_get_device_path_correct_with_attached_disk(self):
         instance_id = self.blockdevice_api.compute_instance_id()
 
         host_device = "/dev/null"
-        virtio = VirtIOClient.from_instance_id(instance_id)
+        tmpdir = FilePath(self.mktemp())
+        tmpdir.makedirs()
+        virtio = VirtIOClient.using_insecure_tls(instance_id, tmpdir)
```

Comment:
It's a good idea, and passing a tmpdir is kind of awkward. But the credential creation code is dependent on how virsh works, particularly the requirement for specific filenames. For this reason, I'll leave it as is.